content/blog/qmstr-newsletter-8.md

```diff
@@ -0,0 +1,120 @@
++++
+title = "Quartermaster v0.1 development update #5: The reporting API and \"Easy Mode\""
+date = "2018-04-09T00:00:00+01:00"
+tags = ["qmstr", "FOSS", "compliance"]
+categories = ["qmstr"]
+banner = "img/blog/qmstr-newsletter-8-title.jpg"
++++
+
+Spring is coming, and so is Quartermaster v0.1! One more sprint to get
+there. Sprint 5 saw the implementation of initial support for Gradle
+based Java projects, a finished definition of the reporting API, the
+introduction of "easy mode" (see below), improved author detection and
+more steps towards an automated HTML reporter. Sprint 5 was the last
+roud of new features before Quartermaster v0.1 will be wrapped up for
+release. We will use the final sprint in this quarter to tie up loose
+ends, polish, containerise, document and demonstrate, and to prepare
+the v0.1 release. The first milestone will be concluded with the v0.2
+requirements workshop on April 11.
+<!--more-->
+
+# Reporting API refactored, implemented in HTML reporter
+
+The elemental workflow from construction to analysis to reporting has
+become more and more defined throughout the development of version
+0.1. In sprint #4 we defined the analysis API, which was a dependency
+for the reporting API. Both are defined as gRPC interfaces to the
+master. This makes module-side implementations in different
+programming languages a possibility, since modules always are run as
+separate processes by the master.
+
+Keeping the administrative effort to run a build in Quartermaster low
+and simple is a key goal for us. With the flexibility of the phased
+workflow toolchain comes some setup effort to configure how the
```

Comment:
...comes with some setup....

Reply:
"With the flexibility of the phased workflow toolchain comes some setup effort" I think is correct.